doc/users/next_whats_new/2019-12-09-legend-textcolor.rst

```diff
@@ -0,0 +1,9 @@
+Legend now has a textcolor kwarg
```

Comment:
More semantic, less technical:
```suggestion
Textcolor for legend labels
```
(+ adapt length of heading markers)

Reply:
done